### DISCUSSION TITLE
requesting a 0.27.3 release of axios

### DISCUSSION THREAD

**Discussion by asselin** [axios/axios#4741] [Archive]
When the error handling was changed in 0.27 (refactoring into AxiosError), we lost stack traces.  The latest published, 0.27.2, has that regression, but it has been fixed in the latest code.

@jasonsaayman Would it be possible to do a 0.27.3 release to bring back stack traces on axios exceptions?

**Comment by DigitalBrainJS** [comment]
We have some breaking changes in the repo, so it can only be v0.28.0 if he decides to do so.

**Comment by jasonsaayman** [comment]
This will be a v1 release, the changes are pretty large and I want to get this as the initial public stable API I will be done with the pre-release this weekend

**Reply by KonradLinkowski** [reply]
Is the stable release date known?

**Comment by jasonsaayman** [comment]
Release is done and the pre-release is up please try it out at axios@next, please leave any feedback on the discussion I have opened
